cities/laupheim.json

```diff
@@ -0,0 +1,209 @@
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88757,
+                    48.23055
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Rosenmarkt",
+                "location": "Schloss Großlaupheim",
+                "opening_hours": null,
+                "opening_hours_unclassified": "Samstag, 17.06.2017"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88359,
+                    48.22969
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Jakobimarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": null,
+                "opening_hours_unclassified": "Donnerstag, 20.07.2017"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88359,
+                    48.22969
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Kirchweihmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": null,
+                "opening_hours_unclassified": "Donnerstag, 12.10.2017"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88941,
+                    48.22771
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Kaltblutpferdemarkt:",
+                "location": "Festplatz Laupheim",
+                "opening_hours": null,
+                "opening_hours_unclassified": "Donnerstag, 12.10.2017"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88359,
+                    48.22969
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Weihnachtsmarkt",
+                "location": "Laupheim Stadtmitte",
+                "opening_hours": null,
+                "opening_hours_unclassified": "Donnerstag, 14.12.2017"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    9.88757,
+                    48.23055
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Weihnachtsmarkt",
+                "location": "Schloss Großlaupheim",
+                "opening_hours": null,
+                "opening_hours_unclassified": "29.11 - 03.12.2017"
```

Comment:
It should actually be possible to express the "unclassified" opening hours in a format that is parseable by the opening_hours.js library. Please take a look at the [evaluation tool](http://openingh.openstreetmap.de/evaluation_tool/). [Chemnitz](https://github.com/wo-ist-markt/wo-ist-markt.github.io/blob/master/cities/chemnitz.json?short_path=bc1363a) specifies there opening hours for time ranges like yours.